package.json

```diff
@@ -36,15 +37,18 @@
     "invariant": "^2.2.4",
     "optimist": "^0.6.1",
     "process": "^0.11.10",
```

Comment:
Could consider dropping this and importing node:process.

Reply:
Based on the readme, I think we want to keep this environment-agnostic (web, node):
> generatable anywhere

I'm happy to commit to making this only node-compatible though. But as the library currently written I believe it'd work in a web environment?